src/main/java/com/sulphate/chatcolor2/listeners/ColorGUIListener.java

```diff
@@ -30,9 +36,13 @@ public void onEvent(InventoryClickEvent event) {
         }
 
         String guiTitle = CC2Utils.colourise(CCStrings.guititle);
-        if (inventoryView.getTitle().equals(guiTitle) && inventoryView.convertSlot(rawSlot) == rawSlot) {
+        if (inventoryView.getTitle().equals(guiTitle)) {
```

Comment:
Does this fix the ability to pick up an item stack from their inventory again?

Reply:
Yes, that regression was fixed for good. The PR in its current state has that bug fixed.